wdl-engine/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Added retry logic for task execution ([#320](https://github.com/stjude-rust-labs/wdl/pull/320)).
 * Added a `Config` type for specifying evaluation configuration ([#320](https://github.com/stjude-rust-labs/wdl/pull/320)).
 * Added progress callback to `WorkflowEvaluator` ([#310](https://github.com/stjude-rust-labs/wdl/pull/310)).
+* Added `chown` workaround to `docker backend` after task execution ([#379](https://github.com/stjude-rust-labs/wdl/pull/379)).
```

Comment:
This should be in the `unreleased` section. I'm assuming this was an artifact of a merge or rebase.